packages/flutter/test/cupertino/date_picker_test.dart

```diff
@@ -1611,9 +1772,12 @@ void main() {
           matchesGoldenFile('date_picker_test.datetime.drag.png'),
         );
       }
-    }, skip: impellerEnabled); // https://github.com/flutter/flutter/issues/143616
+    },
+        skip:
+            impellerEnabled); // https://github.com/flutter/flutter/issues/143616
```

Comment:
The failing analyzer is because of this change. The analyzer is expecting the comment to be on the same line as the `skip`. Could you revert the change that caused all the lines being broken up in this PR? It was this commit, it looks like: https://github.com/flutter/flutter/pull/155260/commits/a3c10f44448e5ed9e646dffc9329bd6b3d58085c

It makes it harder to review and if wanted, could be it's own PR.

Reply:
Oh, my bad! 
I will revert it, thank you!

I'm having some hard times with those formatting because I was unable to fully setup flutter for tests and I'm doing all the changes in "blind" hahah.
Thanks again!